### PR TITLE
Add notices page

### PR DESCRIPTION
**Pull request by lpsinger**
Add a Notices page with links to the GCN Classic notice archives. You can filter the archive using an auto-completing search box.

https://user-images.githubusercontent.com/728407/158174801-f04405ab-c72e-433a-a686-3beff8b5a4fb.mov

### CODE REVIEW

File: app/routes/notices.tsx [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+        >
+          GRBs detected by the GBM instrument on Fermi.
+        </NoticeCard>
+        <NoticeCard
+          name="Fermi LAT"
+          href="https://gcn.gsfc.nasa.gov/fermi_lat_mon_trans.html"
+          tags={['gamma']}
+          selectedTags={tagNames}
+        >
+          GRBs and gamma-ray transients detected by the LAT instrument on Fermi.
```
Comment (jracusin) [inline app/routes/notices.tsx:194]: LAT_mon_trans is actually all non-GRB sources.

File: app/routes/notices.tsx [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+        </NoticeCard>
+        <NoticeCard
+          name="Swift GRBs"
+          href="https://gcn.gsfc.nasa.gov/swift_grbs.html"
+          tags={['gamma', 'x-ray', 'uv', 'optical']}
+          selectedTags={tagNames}
+        >
+          GRBs detected by the Neil Gehrels Swift Observatory.
```
Comment (jracusin) [inline app/routes/notices.tsx:202]: You spell out the full name of Swift here, but not the full name of other observatories.  We should be consistent, either way.

File: app/routes/notices.tsx [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+        </NoticeCard>
+        <NoticeCard
+          name="Swift Sub-Threshold"
+          href="https://gcn.gsfc.nasa.gov/swift_sub_sub_archive.html"
```
Comment (jracusin) [inline app/routes/notices.tsx:206]: These are specifically only BAT, not XRT, so only gamma.

**Comment by lpsinger** [comment]
I think that I addressed all of your comments. Ready for another review.

**Pull request merged**